src/Commands/DisablePackage.php

```diff
@@ -72,11 +72,9 @@ public function handle()
         $this->info('Disabling package '.$this->conveyor->vendor().'\\'.$this->conveyor->package().'...');
         $this->makeProgress();
 
-        // Composer dump-autoload to remove service provider
-        $this->info('Dumping autoloads and undiscovering package...');
-        $this->wrapping->removeFromComposer($this->conveyor->vendor(), $this->conveyor->package());
-        $this->wrapping->removeFromProviders($this->conveyor->vendor(), $this->conveyor->package());
-        $this->conveyor->dumpAutoloads();
```

Comment:
Are you sure the autoloads are updated without this?

Reply:
Yes they are updated automatically each time `composer require` or `composer remove` are run.